doc/doxygen/riot.doxyfile

```diff
@@ -1999,7 +1999,8 @@ PREDEFINED             = DOXYGEN \
                          UART_NUMOF \
                          DEVELHELP \
                          ENABLE_DEBUG \
-                         TEST_SUITES
+                         TEST_SUITES \
+                         MODULE_GNRC_NDP_ROUTER
```

Comment:
Added so I don't have to document the dummy macros (and so that the functions show up in the documentation)